src/archemist/core/optimisation/recipe_converter.py

```diff
@@ -0,0 +1,96 @@
+import pandas as pd
+import yaml
+from archemist.core.persistence.yaml_handler import YamlHandler
+from nested_lookup import nested_update
+from pathlib import Path
+import logging
+import re
+
+class RecipeUpdate:
+    def __init__(self, template_path, recipe_path) -> None:
+        self.recipe_dict  = YamlHandler.load_recipe_file(template_path)
+        self.recipe_dir = recipe_path
+        self.placeholders = []
+        self.placeholder_keys_all = []
+        self.placeholder_keys_any = []
```

Comment:
to be more consistent with the reminder of the library, I suggest turning all of these into internal members, e.g., self._recipe_dict

Reply:
Changed